executer/binary.go

```diff
@@ -0,0 +1,27 @@
+package executer
+
+import (
+	"os/exec"
+
+	"github.com/screwdriver-cd/sd-cmd/api/screwdriver"
+)
+
+// Binary is Binary Executer object
+type Binary struct {
+	Cmd *screwdriver.Command
+	Arg []string
+}
+
+// NewBinary return Binary object
+func NewBinary(cmd *screwdriver.Command, arg []string) (*Binary, error) {
```

Comment:
This should be `New`, or other `New` functions should be `New<type_name>` like `NewBinary`?

Reply:
There will be New, NewBinary, NewDocker and NewHabitat functions. New<type_name> return each type of structure which fulfill Executer interface. New function select the type and return the type of Executer. By doing this main package do not need to choose. Just call executer.New() function.

Reply:
make sense